fiasco/ions.py

```diff
@@ -283,6 +284,25 @@ def formation_temperature(self) -> u.K:
         """
         return self.temperature[np.argmax(self.ioneq)]
 
+    @property
+    def zeta_0(self) -> u.dimensionless_unscaled:
```

Comment:
Is this generally useful or could we just make it private?

Reply:
Likely could be made private.  It might have some useful fringe cases where it would be useful to have access to, but that would be rare.